modules/azure-flexible-server-postgresql/main.tf

```
@@ -1,6 +1,39 @@
 locals {
-  # Let's read the data first
-  data = var.data
-  # Ok let's calculate the subnet_id
-  subnet_id = lookup(local.data.subnet, "name", "NOT_DEFINED") == "NOT_DEFINED" ? local.data.subnet.id : data.azurerm_subnet.subnet[0].id
+  tags = var.tags_from_rg ? data.azurerm_resource_group.resource_group.tags : var.tags
```

```suggestion
  tags = var.tags_from_rg ? merge(data.azurerm_resource_group.resource_group.tags, var.tags) : var.tags
```
See https://github.com/prefapp/tfm/issues/458 and the [terraform merge function](https://developer.hashicorp.com/terraform/language/functions/merge)